### DISCUSSION TITLE
how to upgrade k8s which  create by capi in rancher

### DISCUSSION THREAD

**Discussion by daxiguafan** [rancher/turtles#537] [Q&A]
Can upgrade k8s which was created by  capi  in rancher ui ? If not it there a simple way to do this?

**Comment by alexander-demicev** [comment]
Hi, the UI for the CAPI extension is still in progress. You can upgrade the cluster using any usual methods by changing the resource version directly or with gitops solution.

**Reply by daxiguafan** [reply]
thanks
